src/main/java/moneytransferapp/AccountActivity.java

```diff
@@ -3,12 +3,15 @@
 // @@@SNIPSTART money-transfer-project-template-java-activity-interface
 
 import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
 
 @ActivityInterface
 public interface AccountActivity {
 
+    @ActivityMethod
```

Comment:
not needed. is it added for users? we dont use this annotation in samples either (sample for the other one)

Reply:
Not needed, but I think that it's actually a good idea to use explicitly. Just from the code style and self-documentation perspective.

Reply:
But I will remove it if it's against our common practice.

Reply:
It's fine :)